stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -0,0 +1,105 @@
+package com.stripe.android.paymentsheet
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.test.core.app.ApplicationProvider
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.AbsFakeStripeRepository
+import com.stripe.android.ApiKeyFixtures
+import com.stripe.android.paymentsheet.model.PaymentSelection
+import com.stripe.android.utils.InjectableActivityScenario
+import com.stripe.android.utils.TestUtils.idleLooper
+import com.stripe.android.utils.TestUtils.viewModelFactoryFor
+import com.stripe.android.utils.injectableActivityScenario
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.setMain
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.test.BeforeTest
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+class PaymentSheetActivityTest {
+    @get:Rule
+    val rule = InstantTaskExecutorRule()
```

Comment:
what does this do?

Reply:
it basically sets `viewModelScope = Dispatchers.Immediate`